### PR TITLE
Italian Translation

### PR DESCRIPTION
**Pull request by stefanomarty**
New message translated, couple of minor errors fixed and translation of
row 70 reverted to a more faithful meaning.

### CODE REVIEW

**Comment by kyodev** [comment]
what is the minimum version of php? 5.4 or 5.3?
on [wiki installation: ](https://github.com/PrivateBin/PrivateBin/wiki/Installation)5.3
if 5.4, several translation files must be changed

**Comment by stefanomarty** [comment]
Actually I don't know, I changed to 5.4.0 according to the discussion on issue 201:
https://github.com/PrivateBin/PrivateBin/issues/201#issuecomment-288988380

After my last commit, the new string suggested by @elrido is using a string variable ("%s") instead of 5.4.0, so the minimal version can be injected from a constant. I think this is the best solution.

**Comment by elrido** [comment]
Thanks for pointing this out. It's now 5.4 and we have to update all the docs and other locations that still mention 5.3. I reopened the ticket for this change (#186) and will handle that over there. If you find any other references to 5.3 not listed there, please let me know in that ticket.

File: i18n/it.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
     "en": "it",
     "Paste does not exist, has expired or has been deleted.":
         "Questo messaggio non esiste, è scaduto o è stato cancellato.",
-    "%s requires php 5.3.0 or above to work. Sorry.":
-        "%s richiede PHP 5.3.0 o superiore.",
+    "%s requires php 5.4.0 or above to work. Sorry.":
+        "%s richiede php 5.4.0 o superiore per funzionare. Ci spiace.",
```
Comment (rugk) [inline i18n/it.json]: So as [stated here](https://github.com/PrivateBin/PrivateBin/issues/201#issuecomment-288960981), please replace the PHP version with `%s`.

**Pull request merged**